acs-apiclient/Droid/ExternalFlowWebActivity.cs

```diff
@@ -0,0 +1,67 @@
+﻿
+using System;
+using Android.App;
+using Android.OS;
+using Android.Views;
+using Android.Webkit;
+using acs_apiclient.Android.CustomViews;
+using Android.Widget;
+using Android.Graphics;
+using System.Drawing;
+
+namespace acs_apiclient.Droid
+{
+    [Activity(Label = "ExternalFlowWebActivity", Theme = "@android:style/Theme.NoTitleBar")]
+    public class ExternalFlowWebActivity : Activity
+    {
+        public static string UrlParamKey = "url";
+        private static Android.Graphics.Color ObsidianGrey = Android.Graphics.Color.ParseColor("#222b3c");
```

Comment:
💡 Probably a good candidate for shared code

Reply:
Pulled into `Constants.Colors`

Reply:
✅